airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -213,6 +216,21 @@ def format_url(host: str) -> str:
 
         return host
 
+    def _get_index_patterns(self, ti: TaskInstance | None) -> str:
+        """
+        Get index patterns by calling index_patterns_callable, if provided, or the configured index_patterns.
+
+        :param ti: A TaskInstance object or None.
+        """
+        if self.index_patterns_callable:
+            self.log.debug("Using index_patterns_callable: %s", self.index_patterns_callable)
+            module_path, index_pattern_function = self.index_patterns_callable.rsplit(".", 1)
```

Comment:
We should use `import_string` from airflow.utils.module_loading` for better error handling

Reply:
I have updated it in commit a59826c0a444ddf2bb7c0af84de829c692e7a705